concrete/blocks/autonav/controller.php

```diff
@@ -162,6 +162,11 @@ public function getNavItems($ignore_exclude_nav = false)
             }
         } else {
             $c = $this->collection;
+
+            // let's use the ID of the collection passed in $this->collection
+            if ($this->collection instanceof Page) {
```

Comment:
Can you check against the fully qualified classname rather than the alias? It's `\Concrete\Core\Page\Page`

Reply:
Of course !